docs/_layouts/default.html

```diff
@@ -7,33 +7,40 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no, minimal-ui">
     <link rel="icon" type="image/png" href="https://d2k5nsl2zxldvw.cloudfront.net/images/expensify-favicon.png">
     <link rel="stylesheet" href="/assets/css/styles.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/tocbot/4.1.1/tocbot.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/tocbot/4.12.0/tocbot.css">
+
     <script src="https://kit.fontawesome.com/263e5e8608.js" crossorigin="anonymous"></script>
-    <script defer src="https://cdnjs.cloudflare.com/ajax/libs/tocbot/4.1.1/tocbot.min.js"></script>
+    <script defer src="https://cdnjs.cloudflare.com/ajax/libs/tocbot/4.12.0/tocbot.js"></script>
     <script defer src="/assets/js/main.js"></script>
 </head>
 
 <body>
 
-    <div class="lhn">
-        <img src="/assets/images/expensify-help.svg" class="logo" />
+    <div id="lhn">
+
+        <div class="lhn-header">
+            <div class="header-button" onclick="toggleHeaderMenu()">
+                <i id="angle-up-icon" class="fa-solid fa-angle-up icon hide"></i>
+                <i id="bars-icon" class="fa-solid fa-bars icon"></i>
+            </div>
+            <img src="/assets/images/expensify-help.svg" class="logo" />
+        </div>
 
         <!-- LHN content (Main navigation tree or Article TOC) -->
-        <div class="lhn-content">
+        <div id="lhn-content">
             {% case page.lhn_content %}
             {% when "homepage" %}
-                {% include home-navigation-tree.html %}
+                {% include home-lhn.html %}
             {% when "request-money" %}
-                {% include request-money-navigation-tree.html %}
+                {% include request-money-lhn.html %}
             {% when "send-money" %}
-                {% include send-money-navigation-tree.html %}
+                {% include send-money-lhn.html %}
             {% else %}
-                <div onclick="history.back()" class="main-navigation-tree">
-                    <div class="link">
-                        <i class="fa-solid fa-angle-left caret-icon"></i>Back
-                    </div>
+                <div onclick="navigateBack()" class="icon-with-link">
+                    <i class="fa-solid fa-angle-left icon"></i>
```

Comment:
Probably can be a pointer cursor here too?

![Screen Shot 2022-07-11 at 5 24 13 PM](https://user-images.githubusercontent.com/12268372/178274724-78f067b9-ea42-4398-b751-1da1df875d05.png)

Reply:
👍🏽